src/test/java/com/wikia/webdriver/testcases/blogtests/BlogTests.java

```diff
@@ -82,11 +87,20 @@ public void BlogTests_004_deleteUndelete() {
     String blogTitle = blogPage.getBlogName();
     DeletePageObject deletePage = blogPage.deleteUsingDropdown();
     deletePage.submitDeletion();
-    SpecialRestorePageObject restore = base.getBannerNotifications().clickUndeleteLinkInBannerNotification();
+
+    List<Notification> confirmNotifications = base.getNotifications(CONFIRM_NOTIFICATION);
+    Assertion.assertTrue(confirmNotifications.size()==1,
+            "Number of banner notifications is invalid");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Number of banner notifications is invalid" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)